tests/test_condition_eval.py

```diff
@@ -0,0 +1,129 @@
+import ast
+import pytest
+import sys
+
+from vulture import utils
+from . import check_unreachable
+from . import v
+assert v  # Silence pyflakes
+
+
+def check_condition(code, result):
+    condition = ast.parse(code, mode='eval')
+    assert utils.evaluate_condition(condition) == result
+
+
+def test_false():
+    check_condition('False', False)
+    check_condition('None', False)
+    check_condition("0", False)
+    check_condition("0", False)
+    if sys.version_info > (3, 0):
+        check_condition("1 - 1", False)
+
+
+def test_empty():
+    check_condition("''", False)
+    check_condition("[]", False)
+    check_condition("{}", False)
+
+
+def test_true():
+    check_condition("True", True)
+    check_condition("2", True)
+    check_condition("['foo', 'bar']", True)
+    check_condition("{'a': 1, 'b': 2}", True)
+
+
+def test_errors():
+    with pytest.raises(ValueError) as e:
+        check_condition("foo", False)
+        # checks NameError is handled properly
+        check_condition("foo and False", False)
+        check_condition('__name__ == "__main__"', False)
+        # checks ValueError is handled properly
+        check_condition("False and 1", False)
+        check_condition("False and foo", False)
+        check_condition('chr(-1)', False)
+        # checks AttributeError is handled properly
+        check_condition('getattr(True, "foo")', False)
+        check_condition("hasattr(str, 'foo')", False)
+        # checks TypeError is handled properly
+        check_condition('isinstance(True, True)', False)
+
+        check_condition("not False", False)
```

Comment:
The main advantage of using `eval` over `ast.literal_eval` is we can catch statements with multiple conditions - That's what I think is the main cause when a user might generally neglect an `if` condition.